website/docs/plugin/framework/handling-data/paths.mdx

```diff
@@ -529,3 +530,66 @@ The path which matches the `nested_block_string_attribute` string value in the o
 ```go
 path.Root("root_single_block").AtName("nested_single_block").AtName("nested_block_string_attribute")
 ```
+
+### Building Dynamic Attribute Paths
```

Comment:
Reminder note that here and below may be removable if/when dynamic pathing is removed.

Reply:
Thanks for the reminder, I've adjusted the page in 2f9d877 ⚡